src/gei/HttpClientFactory.cs

```diff
@@ -0,0 +1,20 @@
+using System.Net.Http;
+
+namespace OctoshiftCLI.GithubEnterpriseImporter
+{
+    public class HttpClientFactory : IHttpClientFactory
+    {
+        // todo add the name
+        public HttpClient CreateClient(string name)
+        {
+            #pragma warning disable CA2000 // We don't want to dispose the handler until the client is disposed
+            var handler = new HttpClientHandler
+            {
+                AllowAutoRedirect = false,
+                CheckCertificateRevocationList = true
+            };
```

Comment:
We shouldn't directly create MessageHandlers, that's why we use `HttpClientFactory` in the first place.

Reply:
I think we came to the conclusion to have it work with different kinds of connections, we will keep the factory for now.